sql/core/pom.xml

```diff
@@ -87,6 +87,16 @@
     </dependency>
 
     <dependency>
+      <groupId>org.apache.orc</groupId>
+      <artifactId>orc-core</artifactId>
+      <classifier>${orc.classifier}</classifier>
```

Comment:
In Maven, `classifier` allows to distinguish artifacts that were built from the same POM but differ in their content. Here, `nohive` is used and it refers `orc-core-1.4.0-nohive.jar` instead of `orc-core-1.4.0.jar`.